src/store/ordersActions.js

```diff
@@ -73,6 +77,22 @@ export function getOrder (id) {
     };
 }
 
+export function getOrderForAdministration (adminId) {
+    return dispatch => {
+        fetch(`/api/orders/admin/${adminId}`)
```

Comment:
`/orders/manage/` link would be better, it doesn't make you an admin, just a guy that manages the order at given point in time